docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+    ```
+
+    For example:
+    ```
+    osnadmin channel remove -o HOST1:7081 --channel-id syschannel --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
+    ```
+    When successful you see:
+    ```
+    Status: 204
+    ```
+
+    And when you rerun the `osnadmin channel list` command, you can verify that the system channel was removed:
+    ```
+    osnadmin channel list -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
+    ```
+
+    Examine the output of the command to verify that the system channel was removed:
+
+    ```
+    {
+      "systemChannel": null,
+      "channels":[
+          {
+            "name": "channel1",
+            "url": "/participation/v1/channels/channel1"
+          }
+      ]
+    }
+    ```
+
+    Repeat these commands for each ordering node in the ordering service.
+
+
+### Deploy a new ordering service
+
+Use these steps if you prefer to deploy a new ordering service to try out this feature. Creating the ordering service is a two-step process:
+
+- [Create the ordering organization MSP and generate ordering node certificates](#create-the-ordering-organization-msp-and-generate-ordering-node-certificates)
+- [Configure the orderer.yaml file for each orderer](#configure-the-orderer-yaml-file-for-each-orderer)
+
+#### Create the ordering organization MSP and generate ordering node certificates
+
+Before you can create an ordering service, you need to define the ordering organization MSP definition and generate the TLS and enrollment certificates for each Raft ordering node. To learn how to use a CA to create these identities, check out [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html). After completing that process, you should have the enrollment and TLS certificates for each node as well as the orderer organization MSP definition. To keep track of the generated certificates and MSP you can use the [folder structure](#folder-structure) defined in this topic, although it is not mandatory.
+
+Because this tutorial demonstrates the process for creating a channel with **three orderers** deployed for a single organization, you need to generate enrollment and TLS certificates for each node. Why three orderers? This configuration allows for a policy of `MAJORITY` quorum on the Raft cluster. Namely, when there are three orderers, one at a time can go down for maintenance, while a `MAJORITY` (two of three) is maintained. For more information about the number of nodes you should deploy in production, check out [The Ordering Service](../orderer/ordering_service.html#raft). For simplicity and learning purposes, you have the ability to deploy a single node ordering service, even though such an ordering service will not be highly available and cannot therefore be considered a "production" deployment.
```

Comment:
> Why three orderers? This configuration allows for a policy of `MAJORITY` quorum on the Raft cluster.

This seems to conflate channel config policies and the Raft thesis.  Raft quorum is always majority, regardless of any policy or configuration, this is simply how quorums work.